src/simulation/m_hypoelastic.fpp

```diff
@@ -8,14 +8,15 @@
 module m_hypoelastic
 
     use m_derived_types        !< Definitions of the derived types
-
     use m_global_parameters    !< Definitions of the global parameters
-
+    use m_finite_differences
+    use m_helper
     use m_mpi_proxy            !< Message passing interface (MPI) module proxy
```

Comment:
Could we please remove this line? I think it wasn't present in the previous version.